app/src/main/res/values-ja/strings.xml

```diff
@@ -1,6 +1,5 @@
 <resources>
     <!-- General App strings -->
-    <string name="other_app_icon">App icon of an installed app</string>
```

Comment:
Actually, it will be useful to translate `other_app_icon`, `drawer_open` and `drawer_close` as these are Content Descriptions that accessibility apps such as screen readers use to read out the content on the screen. So users who use such apps can benefit from the translation.

Reply:
Fixed!